cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala

```diff
@@ -97,6 +97,9 @@ object Wadl2Dot {
   val help = parser.flag[Boolean] (List("h", "help"),
                                    "Display usage.")
 
+  val xpathVersion = parser.option[Int](List("t", "xpath-version"), "n",
```

Comment:
The XPath version should have no effect on the DOT file FYI.  But I guess it's okay to have in there for consistency.